_2020/version-control.md

```diff
@@ -428,7 +428,7 @@ index 94bab17..f0013b2 100644
     - Even more reasons to write [good commit messages](https://chris.beams.io/posts/git-commit/)!
 - `git log`: shows a flattened log of history
 - `git log --all --graph --decorate`: visualizes history as a DAG
-- `git diff <filename>`: show differences since the last commit
+- `git diff <filename>`: show changes you made relative to the staging area for the next commit
```

Comment:
"staging area for the next commit" seems redundant. How about just "staging area"?

Reply:
Agree. I have removed the redundant words.